app/src/ep.config.json

```diff
@@ -95,6 +95,23 @@
     "pageId": "<PAGE_ID>",
     "applicationId": "<APP_ID>"
   },
+  "bloomreachSearch": {
+    "enable": false,
+    "config": {
+      "baseUri": "http://core.dxpapi.com/api/v1/core/?",
```

Comment:
@aChanEP We should leave these fields empty OOTB unless we want to allow consumers of this repo to use them as well when starting the app (similar to the asset links). We'll inject them into our reference.elasticpath.com build as part of this ticket https://jira.elasticpath.com/browse/RS-553